pkg/server/server.go

```diff
@@ -2,20 +2,24 @@ package server
 
 import (
 	"bytes"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"path"
+	"strings"
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	rulesspec "github.com/observatorium/api/rules"
 	"github.com/prometheus/prometheus/pkg/rulefmt"
 	"github.com/thanos-io/thanos/pkg/objstore"
+	"gopkg.in/yaml.v3"
 )
 
 const (
-	rulesPath = "/rules.yaml"
+	rulesBasePath = "metrics/rules/"
+	rulesFileName = "/rules.yaml"
```

Comment:
Nit: Do we need the slash? `path.Join` should take care of it.
```suggestion
	rulesFileName = "rules.yaml"
```